src/EnergyPlus/DataEnvironment.hh

```diff
@@ -62,6 +62,16 @@ struct EnergyPlusData;
 
 namespace DataEnvironment {
 
+    enum class GroundTempType
```

Comment:
Made the different GroundTemp field groups into `std::array`s indexed by this `enum`.